### ISSUE TITLE
[BUG]Area songs fail on Servers. Custom Sounds fail on servers. 1.12.2

### ISSUE THREAD

**Issue by Adrinus** [viniciusgf2/Ambience#38]
**Describe the bug**
Placing areas works, and songs can be set up with the config just fine. Some music files seem to just fail to play: leaving a complete silent track. I have had mixed results with some songs seeming to work almost every time, others that only seem to work once in a blue moon.

Beyond this, the server doesn't seem to understand how to load resources and may be why it's impossible to configure new sounds to play from speaker blocks.

**Where playing in SinglePlayer or Multiplayer?**
-Multiplayer

**To Reproduce**
Add a handful of areas, place one and assign a song, place another and assign a different song: the song will fail to play.

**Expected behavior**
The selected song should play in the area selected, specially when instant and play at night are both checked.

**Screenshots**
https://i.gyazo.com/d64d7a0308b3e9e9d4dfe27a411df2e9.png

**Mods used or modpack (please complete the following information):**
Pixelmon 8.0.2

**Please send the followings files to help track the issue:

[ambience properties.zip](https://github.com/viniciusgf2/Ambience/files/5391248/ambience.properties.zip)
[latest.log](https://github.com/viniciusgf2/Ambience/files/5391246/latest.log)

**Comment by Adrinus** [comment]
Did further testing: Version 1.6.1 works on servers for the areas: no luck still on sounds for the speaker.

**Comment by viniciusgf2** [comment]
you have to add the same ambience resource pack from the client in the server for the speaker to work, just create a folder called "resourcepacks" and paste the [AmbienceSounds](https://www.curseforge.com/linkout?remoteUrl=https%253a%252f%252fmega.nz%252ffile%252f3RswGSRC%2523qj7Jn50z30WGgwWgZQRJ-pp1WDNEtNhWWLGcVhMI51g) sound pack inside, but note that if anyone that don't have the exactly same sounds that are in the server sounds pack, it won't connect to the server

**Comment by Adrinus** [comment]
I have ensured that this is the case. I have the same resource pack in both server and client: neither seem to find it: but the area problem is really the big one.

**Comment by viniciusgf2** [comment]
I’ve tested here and found no bug, I’ve created multiple areas and assingo multiple musics to play and it’s working in single and multiplayer, pls double check if your sounds.json is correct and in the ambience musics the areas are created right and everyone on the server have the same configured areas and songs

**Comment by Adrinus** [comment]
The same configuration works perfectly with version 1.6.1: so I know the
sounds work.

**Comment by viniciusgf2** [comment]
have you checked if this file "S:\MultiMC\instances\The-Pixelmon-Modpack-8.0.2\minecraft\resourcepacks\AmbienceSounds\assets\ambience\sounds.json" really exists in the server and in the client?

**Comment by Adrinus** [comment]
Yes. It's there. I tried both via zip, and as a folder just in case.

**Comment by viniciusgf2** [comment]
this is strange, zipped it don't work you must unzip it for the server find the files and register the sounds, but if you you have it unzipped in the right folder I don't know how to help you :(

**Comment by Adrinus** [comment]
When you tested the area sounds: did you use more than one area?

example:

area.One=one
area.Two=two
area.Three=three

If I only use one Area it's fine, but it's after successfully placing "One"
and placing"Two" "Two will not play anything, but "One" still works as
intended.

**Comment by viniciusgf2** [comment]
yes, it works for multiple areas for me without problems

**Comment by Adrinus** [comment]
Man... That *is* bizarre.  All I know is that without changing anything
else, going to version 1.6.1 of your mod makes everything work perfectly,
but all versions after that fail. Any idea what might have changed in the
back-end that we could test?
